content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -37,13 +37,18 @@ these certificates will validate against the cluster root CA.
 
 ## Trusting TLS in a Cluster
 
-Trusting the custom CA from an application running as a pod usually requires
+Trusting the [custom CA](#a-note-to-cluster-administrators) from an application running as a pod usually requires
 some extra application configuration. You will need to add the CA certificate
 bundle to the list of CA certificates that the TLS client or server trusts. For
 example, you would do this with a golang TLS config by parsing the certificate
 chain and adding the parsed certificates to the `RootCAs` field in the
 [`tls.Config`](https://godoc.org/crypto/tls#Config) struct.
 
+{{< note >}}
+If the cluster was setup using kubeadm, by default, the custom CA will be the same as the ConfigMap kube-root-ca.crt
+already placed into the filesystem tree of each container at `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`
```

Comment:
while this happens to be true in some cases, we should *not* document/encourage use of this CA bundle for anything other than verifying connections to the kube-apiserver

Reply:
In my opinion, I would rather have it documented with a disclaimer explaining what you said, including the why and what the alternative/recommended way is (or a link to the existing documentation).

Reply:
> while this happens to be true in some cases, we should not document/encourage use of this CA bundle for anything other than verifying connections to the kube-apiserver

We should not document any additional uses for this configmap. In fact, the configmap itself carries an annotation that explicitly indicates no other usage is provided by the platform: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/certificates/rootcacertpublisher/publisher.go#L44-L45

I could support a doc update that says, "even though this content may be included in the filesystem by kubeadm, it should never be used to verify anything other than the internal endpoints such as the internal service IP or kubernetes.default.svc" (as the description on the configmap itself says).

Reply:
So how about writing it as 

```markdown
Even though the [custom CA](#a-note-to-cluster-administrators) may be included in the filesystem by kubeadm in the ConfigMap kube-root-ca.crt it should never be used to verify anything other than the internal endpoints such as the internal service IP or kubernetes.default.svc. Instead you can distribute the CA certificate as a kubernetes.io/tls Secret that your pods have access to use.
```

Reply:
SIG Auth folks, does that wording work for you? (SIG Docs can finesse the exact phrasing)

I'm not sure myself about using a Secret to distribute a CA cert; personally, I'd use a ConfigMap.

Reply:
@kubernetes/sig-auth-pr-reviews 👋

Reply:
That direction seems reasonable to me (I agree with Tim that I'd use a configmap)